lib/web_ui/lib/src/engine/window.dart

```diff
@@ -305,26 +273,16 @@ class EngineFlutterWindow extends ui.SingletonFlutterWindow {
   /// height: 658 width: 393
   /// height: 368 width: 393
   bool isRotation() {
-    double height = 0;
-    double width = 0;
-    if (domWindow.visualViewport != null) {
-      height =
-          domWindow.visualViewport!.height! * devicePixelRatio;
-      width = domWindow.visualViewport!.width! * devicePixelRatio;
-    } else {
-      height = domWindow.innerHeight! * devicePixelRatio;
-      width = domWindow.innerWidth! * devicePixelRatio;
-    }
-
     // This method compares the new dimensions with the previous ones.
     // Return false if the previous dimensions are not set.
     if (_physicalSize != null) {
+      final ui.Size current = _dimensionsProvider.computePhysicalSize();
```

Comment:
Can you please leave a TODO linking to https://github.com/flutter/flutter/issues/117036. I think we can make this better.